sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -860,7 +860,10 @@ export class ManagementClient extends LinkEntity {
       await defaultLock.acquire(this.managementLock, () => {
         return this._init();
       });
-      await this._mgmtReqResLink!.sendRequest(request);
+      await this._mgmtReqResLink!.sendRequest(request, {
+        delayInSeconds: Constants.defaultDelayBetweenOperationRetriesInSeconds,
+        times: Constants.defaultRetryAttempts
```

Comment:
Even without us passing these values, `updateDisposition()` would use retries right? Is the intention to make it use the `defaultDelayBetweenOperationRetriesInSeconds` instead of `defaultDelayBetweenRetriesInSeconds` ?

What is the criteria for using one value over the other?

Reply:
Good question, keeping inline with defaults in amqp-common, 3 retries with 10 sec timeout and 2 delays of 15 sec adds up to 60 sec overall timeout, so updated to use the defaults accordingly.

Also, no - looks like the original implementation did not have retries on the updateDisposition() since options weren't passed in, and the implementation doesn't set any default except for individual operation timeout.

Reply:
The original implementation does have retries for not just `updateDisposition()` but all the methods in the `ManagementClient` class that use the `this._mgmtReqResLink!.sendRequest()` regardless of whether options are passed in or not.

- [retry being used in sendRequest()](https://github.com/Azure/azure-sdk-for-js/blob/amqp-common/sdk/core/amqp-common/src/requestResponseLink.ts#L219-L229)
- [default values being used by the retry() method](https://github.com/Azure/azure-sdk-for-js/blob/amqp-common/sdk/core/amqp-common/src/retry.ts#L125-L128)

Reply:
Ah, I see - missed the defaults being set within amqp-common. 
Can remove passing in explicit values since they are the same

Reply:
So, we will be reverting the changes to this file?

Reply:
Is there any concern still open from this thread?